ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/Widgets/ComplianceByCluster.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import { Chart, ChartAxis, ChartBar, ChartGroup, ChartLabelProps } from '@patternfly/react-charts';
+
+import { LinkableChartLabel } from 'Components/PatternFly/Charts/LinkableChartLabel';
+import useResizeObserver from 'hooks/useResizeObserver';
+import {
+    CRITICAL_SEVERITY_COLOR,
+    IMPORTANT_HIGH_SEVERITY_COLOR,
+    LOW_SEVERITY_COLOR,
+    MODERATE_MEDIUM_SEVERITY_COLOR,
+} from 'constants/visuals/colors';
+import {
+    defaultChartBarWidth,
+    defaultChartHeight,
+    patternflySeverityTheme,
+} from 'utils/chartUtils';
+
+import WidgetCard from '../../../Dashboard/Widgets/WidgetCard';
+
+const labelLinkCallback = ({ datum }: ChartLabelProps, data: ComplianceByClusterData) => {
+    return typeof datum === 'number' ? data[datum - 1]?.link ?? '' : '';
+};
+
+export type ComplianceByClusterData = {
+    name: string;
+    passing: number;
+    link: string;
+}[];
+
+const mockComplianceData: ComplianceByClusterData = [
+    {
+        name: 'staging',
+        passing: 100,
+        link: '',
+    },
+    {
+        name: 'production',
+        passing: 80,
+        link: '',
+    },
+    {
+        name: 'payments',
+        passing: 73,
+        link: '',
+    },
+    {
+        name: 'patient-charts',
+        passing: 69,
+        link: '',
+    },
+    {
+        name: 'another-cluster',
+        passing: 67,
+        link: '',
+    },
+    {
+        name: 'cluster-name',
+        passing: 39,
+        link: '',
+    },
+];
+
+function ComplianceByCluster() {
+    const [complianceData] = useState(mockComplianceData);
+    const [widgetContainer, setWidgetContainer] = useState<HTMLDivElement | null>(null);
+    const widgetContainerResizeEntry = useResizeObserver(widgetContainer);
+
+    function getBarColor(percent: number): string {
+        if (percent === 100) {
+            return LOW_SEVERITY_COLOR;
+        }
+        if (percent > 50) {
+            return MODERATE_MEDIUM_SEVERITY_COLOR;
+        }
+        if (percent > 25) {
+            return IMPORTANT_HIGH_SEVERITY_COLOR;
+        }
+        return CRITICAL_SEVERITY_COLOR;
+    }
+
+    return (
+        <WidgetCard isLoading={false} header="Compliance by cluster">
+            <div ref={setWidgetContainer}>
+                <Chart
+                    ariaDesc="Compliance coverage percentages by cluster"
+                    ariaTitle="Compliance coverage by cluster"
```

Comment:
Can you search online if whether there is any guideline for or against accessibility attributes when there is heading text?